build-scripts/puddle_conf/test-errata-puddle-3.4.conf

```diff
@@ -0,0 +1,22 @@
+[puddle]
```

Comment:
Most of our directories in the repo uses - instead of underscore. Can we use "puddle-conf" instead of "puddle_conf" ? It just reduces thinking when typing paths.